invenio_cli/helpers/scripts.py

```diff
@@ -101,17 +101,32 @@ def bootstrap(log_config, dev=True, pre=True,
 
 
 @with_appcontext
-def build_assets(log_config):
-    """Build assets."""
+def _build_local_assets(log_config, statics=True, webpack=True):
+    """Build assets in the containers."""
     cli = create_cli()
     runner = current_app.test_cli_runner()
-    # Collect
-    run_command(cli, runner, "collect -v", message="Collecting assets...",
-                verbose=log_config.verbose)
+    if statics:
+        # Collect
+        run_command(cli, runner, "collect -v", message="Collecting assets...",
+                    verbose=log_config.verbose)
+    if webpack:
+        # Build using webpack
+        run_command(cli, runner, "webpack buildall",
+                    message="Building assets...", verbose=log_config.verbose)
```

Comment:
Shelved: It would be nice to break up, dependency (npm) installation and asset building too.